src/app/features/tech-record/components/edit-tech-record-button/edit-tech-record-button.component.ts

```diff
@@ -58,9 +63,16 @@ export class EditTechRecordButtonComponent implements OnInit {
     this.router.navigate([]);
   }
 
+  clickScrollToTop(): void {
+    this.viewportScroller.scrollToPosition([0, 0]);
+  }
+
   submitTechRecord() {
     this.submitCheckFormValidity.emit();
+    this.clickScrollToTop()
 
-    this.toggleEditMode();
+    if (!this.errors) {
```

Comment:
if we're doing this, do we even need to navigate ?

Reply:
Ignore that, we wont get the updated history without navigating to call the refresh, does this still call the refresh then ?

Reply:
you are correct!